### DISCUSSION TITLE
Use a different font for nerd font symbols

### DISCUSSION THREAD

**Discussion by n-p-e** [kovidgoyal/kitty#4566] [Show and tell]
What I have in my `kitty.conf` is:

```
symbol_map U+E0A0-U+E0A3,U+E0C0-U+E0C7 PowerlineSymbols

symbol_map U+E5FA-U+E62B,U+E700-U+E7C5,U+F000-U+F2E0,U+E200-U+E2A9,U+F500-U+FD46,U+E300-U+E3EB,U+F400-U+F4A8,U+2665,U+26A1,U+F27C,U+E0A3,U+E0B4-U+E0C8,U+E0CA,U+E0CC-U+E0D2,U+E0D4,U+23FB-U+23FE,U+2B58,U+F300-U+F313,U+E000-U+E00D [patched font with NF symbols]
```

This eliminates the need for patching your favourite font.

**Comment by page-down** [comment]
This is documented in the FAQ:
https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font

You don't need `[patched font with NF symbols]`, just get the standalone symbol font file. (`Symbols Nerd Font`)

https://github.com/ryanoasis/nerd-fonts/tree/master/src/glyphs
```text
Symbols-1000-em Nerd Font Complete.ttf
```

Then follow the instructions in the FAQ to configure it.

**Reply by n-p-e** [reply]
Cool, didn't know that existed. Thanks for the info!

**Reply by c02y** [reply]
How can I use `Symbols Nerd Font Mono` instead of `Symbols Nerd Font`, I can't manage to do that? `Symbols Nerd Font` icons are too big.

**Reply by zzhaolei** [reply]
On `macOS` I created a `tap`: [formulas](https://github.com/zzhaolei/homebrew-formulas)
